app/experimenter/experiments/forms.py

```diff
@@ -290,9 +290,12 @@ def clean(self):
                     "The size of all branches must add up to 100"
                 ]
 
-        unique_slugs = set([form.cleaned_data["name"] for form in alive_forms])
+        unique_names = set()
+        for form in alive_forms:
+            if form.cleaned_data.get("name"):
+                unique_names.add(form.cleaned_data["name"])
```

Comment:
It is possible to keep this as the list comprehension format, if you want:

```python
unique_slugs = set(form.cleaned_data["name"] for form in alive_forms if form.cleaned_data.get("name"))
```

You might run into some line length issues there though. This is the normal way to split it to two lines:

```python
unique_slugs = set(
	form.cleaned_data["name"] for form in alive_forms
	if form.cleaned_data.get("name")
)
```

Reply:
thanks a lot! I will employ it.